Workbooks/Performance/Single Page Application Insights/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Single Page Application Insights",
+    "description": "Visiualization of Single Page Application Telemetry created by Mezzurite Timing Library",
+    "author": "Microsoft",
+    "tags": ["SPA", "Angular", "React"],
+    "galleries": [{ "type": "workbook", "resourceType": "microsoft.insights/components", "order": 300 }]
```

Comment:
You may want to create an at-scale version of this template and add it to the galleries for Resource Groups and Azure Monitor. At-scale reports summarize the info across multiple App Insights resources (e.g. everything in a sub).